gpflow/experimental/check_shapes/__init__.py

```diff
@@ -85,6 +85,8 @@ def f(...):
     * ``*<name>`` or ``<name>...``, to bind *any* number of dimensions to a variable. Again,
       multiple uses of the same variable name must match the same dimension sizes.
     * ``*`` or ``...``, to allow *any* number of dimensions without constraints.
+    * Any of the above can be prefixed with the keyword ``broadcast``, to allow any value that
+      broadcasts to the specification.
```

Comment:
I think it would be useful to add an example of this feature to the docs.

Reply:
> I think it would be useful to add an example of this feature to the docs.

Fair enough. Done.